spec/request/disability_compensation_form_request_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Disability compensation form', type: :request do
+  include SchemaMatchers
+
+  let(:user) { build(:disabilities_compensation_user) }
+  let(:token) { 'fa0f28d6-224a-4015-a3b0-81e77de269f2' }
+  let(:auth_header) { { 'Authorization' => "Token token=#{token}" } }
+
+  before do
+    Session.create(uuid: user.uuid, token: token)
+    User.create(user)
+  end
+
+  describe 'Get /v0/disability_compensation_form/rated_disabilities' do
+    context 'with a valid evss response' do
```

Comment:
this could be in another PR but we need specs for the http error status codes listed in the EVSS swagger [400, 403, 500] to ensure they're mapping to the right vets-api codes